### PR TITLE
Denize/29 adding quick donations fragment to bottom navigation bar

### PR DESCRIPTION
**Pull request by dignacio0815**

### CODE REVIEW

File: app/src/main/res/layout/fragment_quick_donate.xml [hunk 1]
```diff
@@ -7,18 +7,51 @@
...
+        app:layout_constraintTop_toBottomOf="@+id/tvTitle"
+        app:srcCompat="@drawable/ic_cuppa_24dp" />
+
+    <TextView
+        android:id="@+id/tvTitle"
+        android:layout_width="208dp"
+        android:layout_height="39dp"
+        android:layout_marginTop="80dp"
+        android:gravity="center"
+        android:text="Donate Quickly! "
```
Comment (chiped) [inline app/src/main/res/layout/fragment_quick_donate.xml]: Move the text to `strings.xml`

File: app/src/main/java/com/example/una/fragments/QuickDonateFragment.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.fragment_quick_donate, container, false);
+    }
+
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        ibQuickDonate = view.findViewById(R.id.ibQuickDonate);
```
Comment (chiped) [inline app/src/main/java/com/example/una/fragments/QuickDonateFragment.java]: do you prefer adding multiple `findViewById` callsinstaed of using ButterKnife?

File: app/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -2,4 +2,8 @@
 <resources>
     <dimen name="standard_padding">16dp</dimen>
     <dimen name="small_padding">8dp</dimen>
+    <dimen name="medium_width">120dp</dimen>
```
Comment (chiped) [inline app/src/main/res/values/dimens.xml]: this seems okay for now but soon these names would be confusing. You might want to call it `medium_button_width`. Same for others below

**Comment by chiped** [comment]
since you modified UI, can you add a screenshot?

File: app/src/main/res/values/dimens.xml (hunk 3 above)
Comment (chiped) [reply app/src/main/res/values/dimens.xml]: Not critical to change this now, but I'll leave it up to you

**Comment by dignacio0815** [comment]
QuickDonateFragment

File: app/src/main/res/values/dimens.xml (hunk 3 above)
Comment (dignacio0815) [reply app/src/main/res/values/dimens.xml]: Ok, just updated it

**Pull request merged**